src/main/java/uk/gov/hmcts/divorce/notification/NotificationConstants.java

```diff
@@ -57,14 +57,19 @@
 
     public static final String COURT_EMAIL = "court email";
 
-    public static final String PAPERS_SERVED_ANOTHER_WAY_APPLY = "papersServedAnotherWayApply";
-    public static final String PAPERS_SERVED_ANOTHER_WAY_PARAGRAPH = "papersServedAnotherWayParagraph";
-    public static final String PAPERS_SERVED_ANOTHER_WAY_TITLE = "papersServedAnotherWayTitle";
-
-    public static final String CERTIFICATE = "certificate";
-    public static final String FOREIGN_CERTIFICATE = "foreignCertificate";
-    public static final String FOREIGN_CERTIFICATE_TRANSLATION = "foreignCertificateTranslation";
-    public static final String NAME_CHANGE_PROOF = "nameChangeProof";
+    public static final String PAPERS_SERVED_ANOTHER_WAY = "papersServedAnotherWay";
+    public static final String DIVORCE_SERVED_ANOTHER_WAY = "divorceServedAnotherWay";
+    public static final String SERVE_WIFE_ANOTHER_WAY = "serveWifeAnotherWay";
+    public static final String SERVE_HUSBAND_ANOTHER_WAY = "serveHusbandAnotherWay";
+    public static final String DISSOLUTION_SERVED_ANOTHER_WAY = "dissolutionServedAnotherWay";
+
+    public static final String MISSING_MARRIAGE_CERTIFICATE = "mariageCertificate";
```

Comment:
Can you keep these constants in the class that uses them? If they're used by multiple classes you could extract that logic into a new class

Reply:
done